owncloudApp/src/main/res/values/strings.xml

```diff
@@ -105,16 +105,16 @@
     <string name="pattern_configure_pattern">Enter your pattern</string>
     <string name="pattern_remove_pattern">Remove your pattern</string>
     <string name="pattern_incorrect_pattern">Incorrect pattern</string>
-    <string name="pattern_reenter_pattern">Please re-enter your pattern</string>
-    <string name="pattern_not_same_pattern">The patterns are not same</string>
-    <string name="pattern_error_set">An error occurred while setting the pattern lock</string>
-    <string name="pattern_error_remove">An error occurred while removing the pattern lock</string>
-    <string name="pattern_already_set">Pattern and passcode locks cannot be enabled at the same time, please disable pattern first</string>
-    <string name="passcode_already_set">Pattern and passcode locks cannot be enabled at the same time, please disable passcode first</string>
-    <string name="pattern_no_longer_required">The pattern will not be asked anymore</string>
-    <string name="pattern_configure_your_pattern_explanation">The pattern will be requested every time the app is started</string>
+    <string name="pattern_reenter_pattern">Please re-enter your pattern.</string>
+    <string name="pattern_not_same_pattern">The patterns are not the same.</string>
+    <string name="pattern_error_set">An error occurred while setting the pattern lock.</string>
+    <string name="pattern_error_remove">An error occurred while removing the pattern lock.</string>
+    <string name="pattern_already_set">Pattern and passcode locks cannot be enabled at the same time, please disable pattern first.</string>
+    <string name="passcode_already_set">Pattern and passcode locks cannot be enabled at the same time, please disable passcode first.</string>
+    <string name="pattern_no_longer_required">The pattern will not be asked for anymore.</string>
+    <string name="pattern_configure_your_pattern_explanation">The pattern will be requested every time the app is started.</string>
     <string name="document_provider_locked">Locked</string>
-    <string name="illegal_argument_exception_message">A valid ACTION is needed in the Intent passed to</string>
+    <string name="illegal_argument_exception_message">A valid ACTION is needed in the intent passed to</string>
```

Comment:
In this case with "Intent" we refer to the Kotlin object, which starts with capital letter, I would let it as it was before 😄